editor/export/editor_export_platform_apple_embedded.cpp

```diff
@@ -1908,6 +1911,9 @@ Error EditorExportPlatformAppleEmbedded::_export_project_helper(const Ref<Editor
 		return err;
 	}
 
+	// Generate a unique name for the launch screen to avoid caching.
+	launch_screen_image_file_name = "SplashImage_" + Time::get_singleton()->get_datetime_string_from_system().remove_chars("-T:");
```

Comment:
Might actually be better to use custom image+stretch mode hash to avoid unnecessary changes.